.cirrus.yml

```diff
@@ -0,0 +1,43 @@
+container:
+  image: nixos/nix
```

Comment:
Which container registry does this use?

Reply:
https://hub.docker.com/r/nixos/nix/

Reply:
Oh nice, that's quite a bit better than manually downloading nix as before.